src/index.ts

```diff
@@ -16,23 +16,92 @@ app.use(bodyParser.urlencoded({ extended: false }));
 const db = new MockDB();
 const mariaDB = new MariaDB();
 
-app.get('/', (req, res) => res.send('Hello World!'))
-
-app.listen(port, () => console.log(`Example app listening on ${ip.address()}:${port}!`))
+app.get('/', (req, res) => res.send('Hello World!'));
 
+app.listen(port, () =>
+    console.log(`Example app listening on ${ip.address()}:${port}!`)
+);
 
 app.get('/stories', (req, res) => {
     res.send(JSON.stringify(db.getStories()));
-})
+});
 
 app.get('/review-reservations', (req, res) => {
     res.send(db.getReviewReservations());
-})
+});
+
+app.get('/reviews', (req, res) => {
```

Comment:
change so that it takes a userID and returns the reviews for that user (for the reviews page)
Also needs to be able to take a storyID and return the reviews for that story